Source/WebCore/rendering/style/FillLayer.h

```diff
@@ -87,6 +87,15 @@ class FillLayer : public RefCounted<FillLayer> {
     FillSize size() const { return FillSize(static_cast<FillSizeType>(m_sizeType), m_sizeLength); }
     MaskMode maskMode() const { return static_cast<MaskMode>(m_maskMode); }
 
+    // https://drafts.fxtf.org/css-masking/#the-mask-composite
+    // If there is no further mask layer, the compositing operator must be ignored.
+    CompositeOperator compositeForPainting() const
+    {
+        if (type() == FillLayerType::Mask && !next())
```

Comment:
I think you can safely do:

```suggestion
        ASSERT(type() == FillLayerType::Mask);
        if (!next())
```

The `-webkit-background-composite` property got removed in https://github.com/WebKit/WebKit/commit/a6b4203828773bd332e02365eef5638275d5b29b